### DISCUSSION TITLE
posts/windows-hugo-blog-cloudflare/

### DISCUSSION THREAD

**Discussion by giscus** [leegical/leegical.github.io#10] [Announcements]
# posts/windows-hugo-blog-cloudflare/

使用 Hugo 和 Cloudflare Pages 配置博客，自定义域名并启用SSL，还可以白嫖Cloudflare的CDN。

https://www.haoyep.com/posts/windows-hugo-blog-cloudflare/

**Comment by Nadir-Echo** [comment]
话说博主,好奇怪啊,我搜了好多,有的说cf部署比github pages快很多,有的又说慢,有点心动原本有个动态博客的,想建个静态的,懒得折腾动态的了😂

**Reply by leegical** [reply]
我的博客主要面对国内，而当时建站测试的时候，CF在国内确实比较慢。你建好站后，可以使用阿里云 https://boce.aliyun.com/detect/http 测一下做比较。
本博客目前是放在vercel上，实测比GitHub/cloudflare都快。

**Comment by Nadir-Echo** [comment]
是的喂,而且我看了首页 | Sukka's Blog (skk.moe)这位博主的博客,她是在cf上部署的,速度还挺快的
---- Replied Message ----
***@***.***>Date7/30/2024 ***@***.***>***@***.***>,
***@***.***>Subject[leegical/leegical.github.io] posts/windows-hugo-blog-cloudflare/ (Discussion #10)
posts/windows-hugo-blog-cloudflare/
使用 Hugo 和 Cloudflare Pages 配置博客，自定义域名并启用SSL，还可以白嫖Cloudflare的CDN。
https://www.haoyep.com/posts/windows-hugo-blog-cloudflare/
